.yamato/yamato-config.yml

```diff
@@ -15,7 +15,7 @@ commands:
     - command: |
         linecoverage=$(cat test-results/Report/Summary.xml | grep Linecoverage | grep -Eo '[+-]?[0-9]+([.][0-9]+)?')
         echo "Line coverage: $linecoverage%"
-        if (( $(echo "$linecoverage < 20" | bc -l) )); then exit 1; fi
+        if (( $(echo "$linecoverage < 35" | bc -l) )); then exit 1; fi
```

Comment:
I am curious what this magic number (20/35) is for?

Reply:
just increasing the line coverage threshold for yamato! current line coverage from URDF Importer tests sits around 40% so this is just to raise the bar a bit 

![Screen Shot 2021-08-13 at 11 25 46 AM](https://user-images.githubusercontent.com/31416491/129396972-4395e13f-1b0e-425e-94af-967ef4c08f11.png)